blocks/walls-io/walls-io.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import { createTag } from '../block-helpers.js';
+
+function extractWallParameters($block) {
+  const $parameterContainers = Array.from($block.children);
+
+  const parameters = {
+    src: 'https://walls.io/js/wallsio-widget-1.2.js',
+    async: true,
+    'data-width': '100%',
+    'data-autoheight': 1,
+    'data-lazyload': 1,
+  };
+
+  $parameterContainers.forEach(($parameterContainer) => {
+    const parameterTitle = $parameterContainer.children[0].textContent;
+    const parameterValue = $parameterContainer.children[1].textContent;
+
+    if (parameterTitle === 'Walls.io URL' && parameterValue.includes('https://my.walls.io')) {
```

Comment:
I suggest having the block params be case-insensitive, and since block is already titled "walls-io" that part can be omitted from the param.
```suggestion
    if (parameterTitle.toLowerCase() === 'url' && parameterValue.includes('https://my.walls.io')) {
```